From 551f12f8d6b43fb33c60f919a6d8f68d83e09a46 Mon Sep 17 00:00:00 2001
From: Igor Poboiko <igor.poboiko@gmail.com>
Date: Mon, 22 Feb 2021 14:32:52 +0300
Subject: [PATCH] Fix some dependencies

We no longer depend on Libkdepim, since UiStateSaver was removed from
there and added to KJots source tree.
Bump Qt dependency, since we depend on bunch of libraries with Qt 5.14.0
dependency anyway (e.g. KPimTextEdit)
---
 CMakeLists.txt          | 4 +---
 src/CMakeLists.txt      | 1 -
 src/KJotsSettings.kcfgc | 3 ---
 3 files changed, 1 insertion(+), 7 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3d79806..9fa477e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,10 +24,9 @@ set(KDEPIMLIBS_LIB_VERSION "5.1.0")
 set(KMIME_LIB_VERSION "4.87.0")
 set(KPIMTEXTEDIT_LIB_VERSION "5.14.44")
 set(KONTACTINTERFACE_LIB_VERSION "4.82.0")
-set(LIBKDEPIM_LIB_VERSION "5.14.1")
 
 #Qt Packages
-set(QT_REQUIRED_VERSION "5.6.0")
+set(QT_REQUIRED_VERSION "5.14.0")
 find_package(Qt5 ${QT_REQUIRED_VERSION} CONFIG REQUIRED DBus PrintSupport)
 
 # KF5 Packages
@@ -45,7 +44,6 @@ find_package(KF5Mime ${KMIME_LIB_VERSION} CONFIG REQUIRED)
 find_package(KF5AkonadiNotes ${AKONADINOTES_LIB_VERSION} CONFIG REQUIRED)
 find_package(KF5PimTextEdit ${KPIMTEXTEDIT_LIB_VERSION} CONFIG REQUIRED)
 find_package(KF5KontactInterface ${KONTACTINTERFACE_LIB_VERSION} CONFIG REQUIRED)
-find_package(KF5Libkdepim ${LIBKDEPIM_LIB_VERSION} CONFIG REQUIRED)
 
 find_package(Grantlee5 "5.0" CONFIG REQUIRED)
 
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index a512dab..e66e805 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -49,7 +49,6 @@ target_link_libraries(kjots_common
   KF5::PimTextEdit
   KF5::XmlGui
   KF5::KIOWidgets
-  KF5::Libkdepim
   Grantlee5::Templates
   Grantlee5::TextDocument
 )
diff --git a/src/KJotsSettings.kcfgc b/src/KJotsSettings.kcfgc
index 0db404c..00832d5 100644
--- a/src/KJotsSettings.kcfgc
+++ b/src/KJotsSettings.kcfgc
@@ -3,6 +3,3 @@ File=kjots.kcfg
 ClassName=KJotsSettings
 Singleton=true
 Mutators=true
-# Inherits=KConfigSkeleton
-# IncludeFiles=libkdepim/kpimprefs.h
-# MemberVariables=public
